circleci/utils

```diff
@@ -21,6 +21,29 @@ install_awscli(){
   echo "Completed AWS cli install"
 }
 
+assume_role_with_web_identity() {
+  echo "Setting aws profile to $2"
+  aws configure set profile $2
+  echo "Authenticating via role $1..."
+  aws sts assume-role-with-web-identity \
+    --role-arn "$1" \
+    --role-session-name "oidc-ecr-role-session" \
```

Comment:
We're pushing an approximation for this with the profile name as `$2`; should we leverage that here as well to give more context in CloudTrail logs?